publiccode.yaml

```diff
@@ -0,0 +1,50 @@
+publiccodeYmlVersion: "0.2"
+name: Signalen frontend
+url: "https://github.com/signalen/frontend"
+softwareVersion: "1"
+releaseDate: "2021-05-06"
+platforms:
+  - web
+categories:
+  - it-development
+usedBy:
+  - Gemeente Amsterdam
+developmentStatus: development
+softwareType: standalone/web
+description:
+  nl:
```

Comment:
It says nl, and then we have a mix of English (in the description) and Dutch (in features) following. It would be cool to have it in both Dutch and English, but if not, this should at least be harmonized.